docs/index.md

```diff
@@ -0,0 +1,104 @@
+---
+sidebar_position: 1
+---
+
+# Architecture
+
+## Modules
+
+zkSig is comprised of the following modules:
+
+- **Authentication** - user provides credentials to prove who they are
+- **Encrypted Storage** - used to store encrypted files containing details about agreements and signatures
+- **Public Audit Trail** - a chronological record of agreement events stored in public as hashes
+- **Event Broadcaster** - listens to agreement events and broadcasts messages to smart contracts and webhooks
```

Comment:
Struggling with the way these are worded a little. For consistency maybe:
- **Authentication** - handles credentials to prove who users are
- **Encrypted Storage** - stores encrypted files containing details about agreements and signatures
- **Public Audit Trail** - publicly stores a chronological record of agreement events as hashes
- **Event Broadcaster** - listens to agreement events and broadcasts messages to smart contracts and webhooks